forced-migration/global-oda/js/area.js

```diff
@@ -0,0 +1,47 @@
+$(function() {
+  $('#hcContainer').highcharts({
+    data: {
+      googleSpreadsheetKey: '1P-JsP2LcKwqkTzKtyqmpQGkALbLg3Q59hiI_xZ9jeXw',
+      googleSpreadsheetWorksheet: 1
+    },
+    chart: {
+      zoomType: 'x',
+      type: 'area'
+    },
+    ['#4D6E79', '#303D43', '#907561', '#781F30', '#EC382A', '#61884D', '#C9AC4D', '#52496D', '#5AA992', '#887295', '#2576CE', '#aa266a'],
```

Comment:
This is missing the `colors:` property so the chart doesn't work. Always be sure to check to make sure your chart works when submitting it for review.